test/unit/uk/gov/hmrc/thirdpartyapplication/services/ApplicationServiceSpec.scala

```diff
@@ -117,6 +119,7 @@ class ApplicationServiceSpec extends AsyncHmrcSpec with BeforeAndAfterAll with A
     when(mockEmailConnector.sendApplicationApprovedAdminConfirmation(*, *, *)(*)).thenReturn(successful(response))
     when(mockEmailConnector.sendApplicationApprovedNotification(*, *)(*)).thenReturn(successful(response))
     when(mockEmailConnector.sendApplicationDeletedNotification(*, *, *)(*)).thenReturn(successful(response))
+    when(mockApiPlatformEventService.sendTeamMemberAddedEvent(any[ApplicationData], any[String], any[String])(any[HeaderCarrier])).thenReturn(successful(true))
```

Comment:
Maybe we could add a test where we verify that the service sends this event when a collaborator is added.

Reply:
yep will add a few verifies on the mock for adding a collaborator